tests/Install-Scripts.Test/GivenThatIWantToInstallDotnetFromAScript.cs

```diff
@@ -46,6 +47,8 @@ public class GivenThatIWantToInstallDotnetFromAScript : IDisposable
                 ("release/5.0", "5\\.0\\..*", Quality.None),
                 // Branches are no longer supported starting 6.0, but there are channels that correspond to branches.
                 ("6.0-preview2", "6\\.0\\..*", Quality.Daily | Quality.Signed),
+                ("6.0-preview3", "6\\.0\\..*", Quality.Daily),
```

Comment:
should we test some other cases here, for example preview?

Reply:
No other bits seem to be available in the storage yet. I will ask whether they will be available starting preview 4.